src/include/partitioning/ep-dialogs.rb

```diff
@@ -441,6 +464,14 @@ def MiniWorkflowStepFormatMount(orig_data, creating: false)
       UI.ChangeWidget(Id(:do_mount_attachment), :Enabled, do_mount)
       UI.ChangeWidget(Id(:mount_point), :Value, mount)
 
+      if role == :efi_boot
```

Comment:
As far I see role will be set only for new created partitions (in MiniWorkflowStepRole). So while the update workflow these entries should not be disabled.